packages/parmap/parmap.1.0-rc1/opam

```diff
@@ -26,4 +26,8 @@ depends: [
   "camlp4"
   "ocamlbuild" {build}
   "conf-autoconf"
+  ]
+depexts: [
```

Comment:
since you depend on `conf-autoconf`, you shouldn't need the explicit depexts on autoconf anymore.  `conf-autoconf` also includes support for many other platforms so it should be a superset.